### PR TITLE
Extract new strings (2024-02-14)

### PR DESCRIPTION
**Pull request by mozilla-pontoon**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

**Review by Delphine — COMMENTED** [review]
@adudenamedruby For the two first strings: I want to be sure that these string IDs need to be updated, while the actual content of the strings remains unchanged. Just want to be mindful in these cases, as existing translations are deleted when this happens

**Comment by adudenamedruby** [comment]
cc @jnrahme these strings look like Address autofill strings, and you have context there.

**Comment by adudenamedruby** [comment]
Spoke with @jnrahme and verified what was happening. It was a misunderstanding on how string IDs operate, but yes, we're gonna keep the changes. These strings have not yet been shipped in a user visible way so we're good.

**Comment by Delphine** [comment]
@adudenamedruby The issue I was pointing out was more that we are losing translations that localizers have previously worked on, which is why we try to minimize changing string IDs down the road :) 
In this case, it's just two strings so it's not too much of an issue... but something to keep in mind when thinking about changing string IDs. Thanks!

**Comment by adudenamedruby** [comment]
Yep. We're aware of this. It just wasn't communicated clearly and resulted in a misunderstanding. We do try to avoid this kind of thing pretty actively. :)

File: templates/firefox-ios.xliff [hunk 1]
```diff
@@ -4044,14 +4106,34 @@ Private Tab</source>
...
+        <source>Get suggestions from %@ related to your search</source>
+        <note>In the Search page of the Settings menu, the description for the setting to enable Suggestions from Firefox. Placeholder is for the app name - Firefox.</note>
+      </trans-unit>
       <trans-unit id="Settings.Search.Suggest.ShowNonSponsoredSuggestions.Title.v124" xml:space="preserve">
         <source>Suggestions from %@</source>
         <note>In the Search page of the Settings menu, the title for setting to enable Suggestions from Firefox. Placeholder is for the app name - Firefox.</note>
       </trans-unit>
+      <trans-unit id="Settings.Search.Suggest.ShowNonSponsoredSuggestions.Title.v124.v2" xml:space="preserve">
+        <source>Suggestions from the Web</source>
+        <note>In the Search page of the Settings menu, the title for setting to enable Suggestions from Firefox.</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:4135]: Maybe nit: should the comment rather say "(...)the title for setting to enable Suggestions from the Web" vs Firefox?

**Review by Delphine — COMMENTED** [review]
Just one question, otherwise seems good to go! @adudenamedruby

File: templates/firefox-ios.xliff (hunk 1 above)
Comment (adudenamedruby) [reply templates/firefox-ios.xliff:4135]: Addressed! If you extract again, the update will appear 🪄  :D

**Review by Delphine — APPROVED** [review]

**Pull request merged**